src/client/src/Stream.h

```diff
@@ -78,12 +78,25 @@ extern "C" {
  */
 #define MAX_BLOCKING_PUT_WAIT 15 * HUNDREDS_OF_NANOS_IN_A_SECOND
 
+/**
+ * Maximum about of "tags" elements allowed in fragment metadata
+ */
+#define MAX_TAGS_COUNT 10
```

Comment:
Does this not conflict with other defines? Consider adding more verbosity to this